tests/validation/tests/v3_api/test_custom_host_reg.py

```diff
@@ -47,7 +47,8 @@ def test_deploy_rancher_server():
     if "v2.5" in RANCHER_SERVER_VERSION or \
             "master" in RANCHER_SERVER_VERSION or \
             "v2.6" in RANCHER_SERVER_VERSION or \
-            "v2.7" in RANCHER_SERVER_VERSION:
+            "v2.7" in RANCHER_SERVER_VERSION or \
```

Comment:
Should we keep 2.5 ?

Reply:
I wrote a task to instead use a regex. For now, I think this is fine since we need it ASAP for our release testing.

Reply:
https://github.com/rancher/qa-tasks/issues/997

Reply:
@izaac